plugins/dell-dock/fu-dell-dock-hid.c

```diff
@@ -71,12 +71,14 @@ fu_dell_dock_hid_set_report (FuDevice *self,
 		    G_USB_DEVICE_RECIPIENT_INTERFACE, HID_REPORT_SET, 0x0200,
 		    0x0000, outbuffer, 192, &actual_len,
 		    HIDI2C_TRANSACTION_TIMEOUT, NULL, &error_local);
-		if (ret)
+		if (ret) {
 			break;
```

Comment:
I don't think the break needs to be in the if clause; the break is basically like a "goto out"

Reply:
Where else would it be?  The loop does need to be broken out of; could you suggest what you have in mind?

Reply:
i think:

    if (ret)
        break;

is the right way to do it.

Reply:
OK, so pretty much what I had before :)
Adjusted.